spec/models/course_user_spec.rb

```diff
@@ -119,6 +119,27 @@
       end
     end
 
+    describe '#level_number' do
+      subject { student.level_number }
+      before do
+        create :course_level, course: course, experience_points_threshold: 100
+        create :course_level, course: course, experience_points_threshold: 200
+      end
+
+      context 'when student has no experience points' do
+        it { is_expected.to eq 0 }
+      end
+
+      context 'after experience points have been awarded to student' do
+        before do
+          create :course_experience_points_record, points_awarded: 150, course_user: student
+        end
+        it 'levels up' do
```

Comment:
Don't mix is_expected with an explicit message. Do:
```ruby
it { is_expected.to eq(1) }
```
Also, function calls should be parenthesised, unless it is a DSL.